### PR TITLE
wip release notes for 0.4.14

### PR DESCRIPTION
**Pull request by alisterburt**
# Description
Work in progress release notes for `0.4.14`

not sure if I've put these in the right place - file was gitignored but I added it because I couldn't see where else it should go

### CODE REVIEW

File: docs/release/release_0_4_14.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# napari 0.4.14
+
+We're happy to announce the release of napari 0.4.14!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+This is a small release containing some new features and fixing a few issues that have come up since `0.4.13` was released.
```
Comment (andy-sweet) [inline docs/release/release_0_4_14.md]:
```suggestion
This is a small release containing some new features and fixing a few issues that have come up since 0.4.13 was released.
```

I think we should be consistent about whether version numbers are in backticks. Version 0.4.14 is not above and I think backticks are mostly reserved for code, so I suggest removing them here.

File: docs/release/release_0_4_14.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- fix about on python 3.10 (#3972)
+- Revert changes to scikit-image test API (#3979)
+- Fix missing builtins in bundle (#3982)
+- don't use texture_format auto if float textures not available on machine (#3990)
+- Fix error when setting ndisplay=3 with empty shapes layer in viewer (#4003)
+- Fix bug preventing close button on dock widgets (#4006)
+
+## API Changes
+
+## Deprecations
```
Comment (andy-sweet) [inline docs/release/release_0_4_14.md:45]: Just as a reminder, if #4007 gets in, then it probably contributes an API change (`out_of_slice_display` is a new param/property/event) and a deprecation (`n_dimensional` is deprecated).

File: docs/release/release_0_4_14.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+We're happy to announce the release of napari 0.4.14!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+This is a small release containing some new features and fixing a few issues that have come up since `0.4.13` was released.
+
+This release adds the ability to convert shapes layers to labels layers in a context menu available on right clicking the layer (#3978). Our vectors layer has gained the
```
Comment (andy-sweet) [inline docs/release/release_0_4_14.md]:
```suggestion
This release adds the ability to convert a shapes layer to a labels layer from a context menu available on right clicking the layer list entry (#3978). Our vectors layer has gained the 
```

Optional: I found the initial wording a little hard to parse. Consider the suggestion, but feel to ignore.

**Review by andy-sweet — APPROVED** [review]
Looks good to me. Line comments are optional suggestions and reminders.

File: docs/release/release_0_4_14.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+layer can now be hidden individually (#3625).
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+Complete list of changes below:
+
+## Highlights
+- Enable shapes layer (right click) > convert to labels (#3978)
+- out-of-slice rendering for Vectors (#2902)
```
Comment (nclack) [inline docs/release/release_0_4_14.md]: The listed changes lead with a mix of upper and lower case. Other change logs seem to consistently lead with uppercase.

**Review by nclack — APPROVED** [review]
Thanks! Left one comment about sentence casing.

**Review by tlambert03 — APPROVED** [review]

File: docs/release/release_0_4_14.md (hunk 2 above)
Comment (jni) [reply docs/release/release_0_4_14.md:45]: Note: API Changes are reserved for *existing APIs that we break*. *Addition* of parameters/attributes are not API changes.

**Comment by alisterburt** [comment]
Thanks @andy-sweet @nclack @tlambert03 

It looks like we're waiting on #4007 which is getting close - let's merge here and I'll leave a note there to update the release notes prior to release

File: docs/release/release_0_4_14.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
```
Comment (alisterburt) [inline docs/release/release_0_4_14.md]:
```suggestion
- Out-of-slice rendering for Vectors (#2902)
```

File: docs/release/release_0_4_14.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+- Hide points individually (#3625)
+
+## New Features
+
+## Improvements
+- Update tests to remove warnings (#3974)
+- add test to check that things are removed properly (#3996)
```
Comment (alisterburt) [inline docs/release/release_0_4_14.md]:
```suggestion
- Add test to check that things are removed properly (#3996)
```

File: docs/release/release_0_4_14.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+- Change base point slice thickness to half a unit (#3997)
+- Change base vector slice thickness to half a unit (#4001)
+
+## Bug Fixes
+- fix about on python 3.10 (#3972)
```
Comment (alisterburt) [inline docs/release/release_0_4_14.md]:
```suggestion
- Fix about on python 3.10 (#3972)
```

File: docs/release/release_0_4_14.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+- Revert changes to scikit-image test API (#3979)
+- Fix missing builtins in bundle (#3982)
+- don't use texture_format auto if float textures not available on machine (#3990)
```
Comment (alisterburt) [inline docs/release/release_0_4_14.md]:
```suggestion
- Don't use texture_format auto if float textures not available on machine (#3990)
```

**Review by alisterburt — COMMENTED** [review]
fixed casing suggested by @nclack

File: docs/release/release_0_4_14.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+- Fix bug preventing close button on dock widgets (#4006)
+
+## API Changes
+
+## Deprecations
+
+
+## Build Tools and Docs
+- DOC: Misc doc syntax. (#3985)
+
```
Comment (alisterburt) [inline docs/release/release_0_4_14.md]:
```suggestion
- Fix include_package_data inclusion in the "your first plugin" docs page (#4022)
```

File: docs/release/release_0_4_14.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+- Hide points individually (#3625)
+
+## New Features
+
+## Improvements
+- Update tests to remove warnings (#3974)
+- Add test to check that things are removed properly (#3996)
+- Change base point slice thickness to half a unit (#3997)
+- Change base vector slice thickness to half a unit (#4001)
+
```
Comment (alisterburt) [inline docs/release/release_0_4_14.md:34]:
```suggestion
- Use scikit-image[data] in the bundle (#4024)

```

File: docs/release/release_0_4_14.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+## Deprecations
+
+
+## Build Tools and Docs
+- DOC: Misc doc syntax. (#3985)
+
+
+## 5 authors added to this release (alphabetical)
+- [Alister Burt](https://github.com/napari/napari/commits?author=alisterburt) - @alisterburt
+- [Lorenzo Gaifas](https://github.com/napari/napari/commits?author=brisvag) - @brisvag
```
Comment (alisterburt) [inline docs/release/release_0_4_14.md:56]:
```suggestion
- [Jaime Rodríguez-Guerra](https://github.com/napari/napari/commits?author=jaimergp) - @jaimergp
- [Lorenzo Gaifas](https://github.com/napari/napari/commits?author=brisvag) - @brisvag
```

File: docs/release/release_0_4_14.md (hunk 2 above)
Comment (alisterburt) [reply docs/release/release_0_4_14.md:45]: just to check @andy-sweet - `n_dimensional` is not deprecated now, right?

File: docs/release/release_0_4_14.md [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+- [Alister Burt](https://github.com/napari/napari/commits?author=alisterburt) - @alisterburt
+- [Jaime Rodríguez-Guerra](https://github.com/napari/napari/commits?author=jaimergp) - @jaimergp
+- [Lorenzo Gaifas](https://github.com/napari/napari/commits?author=brisvag) - @brisvag
+- [Matthias Bussonnier](https://github.com/napari/napari/commits?author=Carreau) - @Carreau
+- [Pam](https://github.com/napari/napari/commits?author=ppwadhwa) - @ppwadhwa
+- [Talley Lambert](https://github.com/napari/napari/commits?author=tlambert03) - @tlambert03
+
+
+## 8 reviewers added to this release (alphabetical)
+- [Alister Burt](https://github.com/napari/napari/commits?author=alisterburt) - @alisterburt
```
Comment (alisterburt) [inline docs/release/release_0_4_14.md:63]:
```suggestion
- [Alister Burt](https://github.com/napari/napari/commits?author=alisterburt) - @alisterburt
- [Andy Sweet](https://github.com/napari/napari/commits?author=andy-sweet) - @andy-sweet
```

File: docs/release/release_0_4_14.md [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+
+## New Features
+
+## Improvements
+- Update tests to remove warnings (#3974)
+- Add test to check that things are removed properly (#3996)
+- Change base point slice thickness to half a unit (#3997)
+- Change base vector slice thickness to half a unit (#4001)
+- Use scikit-image[data] in the bundle (#4024)
+
```
Comment (alisterburt) [inline docs/release/release_0_4_14.md:34]:
```suggestion
- Rename n_dimensional to out_of_slice_display in Points and Vectors (#4007)

```

File: docs/release/release_0_4_14.md (hunk 2 above)
Comment (andy-sweet) [reply docs/release/release_0_4_14.md:45]: It's deprecated/discouraged in the docstrings, but we don't issue deprecation warnings. So yeah, I think we should omit it here until those warnings are added.

Comment (alisterburt) [reply docs/release/release_0_4_14.md:45]: Perfect, thanks!

File: docs/release/release_0_4_14.md [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+## Deprecations
+
+
+## Build Tools and Docs
+- DOC: Misc doc syntax. (#3985)
+- Fix include_package_data inclusion in the "your first plugin" docs page (#4022)
+
+## 5 authors added to this release (alphabetical)
+- [Alister Burt](https://github.com/napari/napari/commits?author=alisterburt) - @alisterburt
```
Comment (alisterburt) [inline docs/release/release_0_4_14.md:53]:
```suggestion
- [Alister Burt](https://github.com/napari/napari/commits?author=alisterburt) - @alisterburt
- [Andy Sweet](https://github.com/napari/napari/commits?author=andy-sweet) - @andy-sweet
```
